open_seq2seq/models/model.py

```diff
@@ -150,7 +153,8 @@ def __init__(self,
           name_or_scope=tf.get_variable_scope(),
           reuse=force_var_reuse,
           initializer=initializer,
-          dtype=self.get_dtype(),
+          dtype=tf.float16 if self.params['dtype'] == "mixed"
```

Comment:
I like get_dtype() function better. Later, we might have more types, say for inference or different variants of "mixed"